kytos/core/exceptions.py

```diff
@@ -86,7 +86,52 @@ class KytosLinkCreationError(Exception):
 
 
 class KytosTagtypeNotSupported(Exception):
-    """Exception thronw when a not supported tag type is not supported"""
+    """Exception thrown when a not supported tag type is not supported"""
```

Comment:
Added `KytosTagError`

Reply:
Nice. Can you also add it as a base class of `KytosTagsNotInTagRanges` and `KytosTagsNotInTagRanges`? That way, if clients don't want to handle specifics, since all exceptions have great names and their `__repr__` is also great, they can just end up handling `KytosTagError`. As you're doing on the NApps it's great, but let's also provide this option to keep the hierarchy of exceptions complete for the related exceptions.

Reply:
Added them.

Reply:
Looks good.